sycl/doc/GettingStartedAlveo.md

```diff
@@ -0,0 +1,449 @@
+Getting started with SYCL with a Xilinx FPGA U200 Alveo board and Ubuntu 19.04
+==============================================================================
+
+We assume that you have the latest Ubuntu 19.04 version installed on
+an `x86_64` machine.
+
+It might work for some other version of Ubuntu or Debian with some
+adaptations.
+
+
+## Installing the Alveo U200 board
+
+If you do not have a real board and wants to use only software or
+hardware emulation, just skip this section.
+
+Install an Alveo U200 board in the machine with the right cooling.
+The PCIe auxiliary power is not necessary for simple test not using the full
+power of the board.
+
+
+### Use a modern BIOS
+
+Update the BIOS of your machine to the latest version. The Alveo U200
+board might not be detected by `lspci` at the PCIe level if you do not
+have a good BIOS.
+
+If you are running Linux on top of an EFI BIOS, you can probably use
+the firmware capsule concept and try:
+```bash
+# Install the firmware manager and daemon
+sudo apt install fwupdate fwupd
+# Refresh the list of available firmware
+sudo fwupdmgr refresh
+# Do the firmware update if any
+sudo fwupdmgr update
+```
+
+If you are not running an EFI BIOS, you can follow the manual recipe
+of the BIOS update for your the motherboard. Typically look for the
```

Comment:
"follow the manual recipe of the BIOS update for your the motherboard" -> "follow the manual BIOS update recipe for your motherboard"